### PR TITLE
Add Desk.cpl

### PR DESCRIPTION
**Pull request by LuxNoBulIshit**
Launch an scr payload by calling InstallScreenSaver

### CODE REVIEW

File: yml/OSLibraries/desk.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    Privileges: User
+    MitreID: T1218.011
+    OperatingSystem: Windows 10, Windows 11
+Full_Path:
+  - Path: C:\Windows\System32\desk.cpl
+  - Path: C:\Windows\SysWOW64\desk.cpl
+  - Path: C:\Windows\WinSxS\amd64_microsoft-windows-desk_31bf3856ad364e35_10.0.19041.1_none_6273e6cab85afd8c\desk.cpl
+  - Path: C:\Windows\WinSxS\wow64_microsoft-windows-desk_31bf3856ad364e35_10.0.19041.1_none_6cc8911cecbbbf87\desk.cpl
+Detection:
+  - IOC: conhost.exe spawning unexpected processes
```
Comment (wietze) [inline yml/OSLibraries/desk.yml:20]: This is probably not updated after copy/pasting

File: yml/OSLibraries/desk.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Resources:
+  - Link: https://twitter.com/VakninHai/status/1517027824984547329
+  - Link: https://file.io/x3CRqMdX95i0
```
Comment (wietze) [inline yml/OSLibraries/desk.yml:23]: What kind of file is this,  is this a sample `.scr` file? If so, could you please put the source code on GitHub and link to that instead?

File: yml/OSLibraries/desk.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+Name: desk.cpl
+Description: Display Propertieslibary
+Author: Hai vaknin
+Created: 2022-04-21
+Commands:
+  - Command: "rundll32.exe desk.cpl,InstallScreenSaver C:\temp\file.scr"
```
Comment (wietze) [inline yml/OSLibraries/desk.yml:7]: Aren't `.scr` files regular PE files (i.e. `.exe`) with a different extension? I was hoping the following would work, but it doesn't seem to:
```
rundll32 desk.cpl,InstallScreenSaver c:\windows\system32\calc.exe
```
Is this expected? If so, can you elaborate on what the requirements for the `.scr` file are?

File: yml/OSLibraries/desk.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    Usecase: Launch an screen saver file.
+    Category: Execute
+    Privileges: User
+    MitreID: T1218.011
+    OperatingSystem: Windows 10, Windows 11
+Full_Path:
+  - Path: C:\Windows\System32\desk.cpl
+  - Path: C:\Windows\SysWOW64\desk.cpl
+  - Path: C:\Windows\WinSxS\amd64_microsoft-windows-desk_31bf3856ad364e35_10.0.19041.1_none_6273e6cab85afd8c\desk.cpl
+  - Path: C:\Windows\WinSxS\wow64_microsoft-windows-desk_31bf3856ad364e35_10.0.19041.1_none_6cc8911cecbbbf87\desk.cpl
```
Comment (wietze) [inline yml/OSLibraries/desk.yml:18]: These paths change regularly and can change from system to system,  so let's omit them from the entry.

**Review by wietze — CHANGES_REQUESTED** [review]
Nice find, few questions/remarks from me.

File: yml/OSLibraries/desk.yml (hunk 2 above)
Comment (LuxNoBulIshit) [reply yml/OSLibraries/desk.yml:23]: https://file.io/x3CRqMdX95i0

here you can find the scr file that execute calculator .

Comment (wietze) [reply yml/OSLibraries/desk.yml:23]: Can you upload the source code to a public space such as GitHub and link to that instead please?

File: yml/OSLibraries/desk.yml (hunk 3 above)
Comment (LuxNoBulIshit) [reply yml/OSLibraries/desk.yml:7]: Change the calc.exe to calc.scr and run it 👍 

its will work ( just did ).

File: yml/OSLibraries/desk.yml (hunk 2 above)
Comment (LuxNoBulIshit) [reply yml/OSLibraries/desk.yml:23]: Change the calc.exe to calc.scr and run it 👍

its will work ( just did ).

File: yml/OSLibraries/desk.yml (hunk 4 above)
Comment (LuxNoBulIshit) [reply yml/OSLibraries/desk.yml]: totally agree 👍

File: yml/OSLibraries/desk.yml (hunk 3 above)
Comment (LuxNoBulIshit) [reply yml/OSLibraries/desk.yml:7]: @wietze 

did you manage to make it work ?

**Review by wietze — APPROVED** [review]

**Comment by wietze** [comment]
After a few final tweaks, it has now been merged into the project. Thank you for your contribution!

**Pull request merged**